src/main/java/properties/ReadProperties.java

```diff
@@ -43,12 +44,25 @@ private void readPropertyFile(String file) {
     }
 
     /***
-     * Receives the key value from config.properties
-     * @param key key from config.properties
+     * Receives the key value from property files
+     * @param key key from property files
      * @return key value
      */
-    public String getKeyProperty(String propsFileName, String key) {
-        readPropertyFile(propsFileName);
-        return propsMap.get(propsFileName).getProperty(key);
+    public String getKeyProperty(String environment, String key) {
+        switch (environment) {
```

Comment:
Are you sure you need switch here? 
in case of DEV env you need application-dev.properties file, in case of QA you need application-qa.properties file and so on, so your filename CONTAINS environment name but written in lowercase. 
Can you use application-%s.properties as pattern and format it according to selected environment?

Reply:
Understood and fixed